applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -96,6 +96,50 @@ void CrBeamElement3D2N::GetDofList(DofsVectorType& rElementalDofList,
     }
 }
 
+void CrBeamElement3D2N::Initialize(const ProcessInfo& rCurrentProcessInfo)
+{
+
+    KRATOS_TRY
+
+    mIsInitialization = true;
+
+    KRATOS_CATCH("")
+}
+
+void CrBeamElement3D2N::InitializeSolutionStep(const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    if (mIsInitialization)
+    {
+        if (rCurrentProcessInfo.Has(RESET_DISPLACEMENTS))
+        {
+            if (rCurrentProcessInfo[RESET_DISPLACEMENTS])
+                noalias(mInternalForcesFinalizedPrevious) = mInternalForcesFinalized;

```

Comment:
I agree with @philbucher 's comments in the original PR. 

we shouldn't store a full vector with the internal forces unless it is completely needed...

in my opinion the clean way of having this would be through the mechanism of the initial state that was recently added by @AlejandroCornejo.

that is, one may store the initial moment and axial force in the initial state and just use it to compute the initial forces when needed. That way it would be exactly identical to what is doen for 2D and 3D volume elements